patches/@agoric+smart-wallet+0.5.3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@agoric/smart-wallet/src/marshal-contexts.js b/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
+index de5b3b9..b7594b6 100644
+--- a/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
++++ b/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
+@@ -286,7 +286,7 @@ export const makeImportContext = (makePresence = defaultMakePresence) => {
+      * @param {string} iface
+      */
+     fromBoard: (slot, iface) => {
+-      isDefaultBoardId(slot) || Fail`bad board slot ${q(slot)}`;
++      isDefaultBoardId(slot) || slot === null || Fail`bad board slot ${q(slot)}`;
```

Comment:
how do we stop patching this?

This app only needs the board marshaller, not the whole import context the smart-wallet itself needs.

Would this work? https://github.com/Agoric/agoric-sdk/blob/c6d8a278ad7d7c0ad758f57c7e4db858d6a67e53/packages/internal/src/marshal.js#L32-L41

Reply:
Yes, I think that will work, but while working on [productizing the bidding CLI](https://github.com/Agoric/agoric-sdk/pull/7939), I realized that relying on remotables to know their slot/boadID is awkward.

Also, the `BoardRemote${nonalleged}` idiom is over-kill. IMO it's better, conceptually, to just use `${nonalleged}` as the debug name. But more importantly: users pay by the byte when they submit offers.

The way I've been teaching it lately is to use a typical translation table, parameterized by what to do when you get a cache miss...

https://github.com/Agoric/documentation/blob/daf6472eaa257bb28c78341f521a87fd684bcd99/snippets/test-marshal.js#L97-L121

And the board slotting marshaller synthesizes a remotable when `slotToVal` has a cache miss:

https://github.com/Agoric/documentation/blob/daf6472eaa257bb28c78341f521a87fd684bcd99/snippets/test-marshal.js#L319C1-L326C81

Reply:
Okay I  got something like Dan's example working